app/models/user.rb

```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+    def admin?
+      role == "admin"
+    end
```

Comment:
You should add some validations for the role here: probably a presence validation, plus an inclusion validation to make sure it's one of the valid role types

Reply:
You should also add tests for this!